backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -27,6 +27,16 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
+include::modules/migration-debugging-velero-admission-webhooks.adoc[leveloffset=+1]
```

Comment:
You cannot include this file here. Nested assemblies are not allowed, include statements are not allowed in modules, and assemblies are not allowed in the modules directory.

If I remove this include statement and use the `_topic_map.yml` file that I described, the build passes.